lib/run-status.js

```diff
@@ -134,7 +134,7 @@ RunStatus.prototype.prefixTitle = function (file) {
 	var separator = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
 
 	var prefix = path.relative('.', file)
-		.replace(this.base, '')
+		.replace(new RegExp('^' + this.base), '')
```

Comment:
You need to escape the value here.

https://github.com/sindresorhus/escape-string-regexp